tutorials/Bert_SQUAD_Interpret.ipynb

```diff
@@ -0,0 +1,1111 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we pretrained the model, we can loading the tokenizer and pre-trained BERT model using commands described below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# replace <PATH-TO-SAVED-MODEL> with the real path of the saved model\n",
+    "\n",
+    "model_path = <PATH-TO-SAVED-MODEL>\n",
+    "\n",
+    "# load model\n",
+    "model = BertForQuestionAnswering.from_pretrained(model_path) #, output_attentions=True)\n",
+    "model.to(device)\n",
+    "model.eval()\n",
+    "model.zero_grad()\n",
+    "\n",
+    "# load tokenizer\n",
+    "tokenizer = BertTokenizer.from_pretrained(model_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A helper function to perform forward pass of the model and accomplish predictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def predict(inputs, token_type_ids=None, position_ids=None, attention_mask=None):\n",
+    "    return model(inputs, token_type_ids=token_type_ids,\n",
+    "                 position_ids=position_ids, attention_mask=attention_mask, )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Defining a custom forward function that will allow us to access the start and end postitions of our prediction using `position` input argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def squad_pos_forward_func(inputs, token_type_ids=None, position_ids=None, attention_mask=None, position=0):\n",
+    "    pred = predict(inputs,\n",
+    "                   token_type_ids=token_type_ids,\n",
+    "                   position_ids=position_ids,\n",
+    "                   attention_mask=attention_mask)\n",
+    "    pred = pred[position]\n",
+    "    return pred.max(1).values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's interpret parent `BertEmbeddings` layer. For that reason we separate embedding layer and precompute the embeddings. `configure_interpretable_embedding_layer` function helps us to do that. It returns `InterpretableEmbeddingBase` layer that wraps `BertEmbeddings` and can be used to access the embedding vectors. Note that we need to remove `InterpretableEmbeddingBase` wrapper from our model using `remove_interpretable_embedding_layer` function after we finish interpretation.\n",
+    "\n",
+    "Note that there is a way to interpret BertEmbeddings without configuring interpretable embedding layers that we will make available soon, However it is important to not that configuring interpretable embedding layers gives us more flexibility."
```

Comment:
Nit: `it is important to not that configuring` -> `it is important to note that configuring`